src/JsonApi/Serializer/ReservedAttributeNameConverter.php

```diff
@@ -13,14 +13,15 @@
 
 namespace ApiPlatform\Core\JsonApi\Serializer;
 
+use Symfony\Component\Serializer\NameConverter\AdvancedNameConverterInterface;
 use Symfony\Component\Serializer\NameConverter\NameConverterInterface;
 
 /**
  * Reserved attribute name converter.
  *
  * @author Baptiste Meyer <baptiste.meyer@gmail.com>
  */
-final class ReservedAttributeNameConverter implements NameConverterInterface
+final class ReservedAttributeNameConverter implements AdvancedNameConverterInterface
```

Comment:
What about  `InnerFieldsNameConverter`?

Reply:
Not sure to get what you want me to change :p.